README.md

```diff
@@ -97,326 +90,75 @@ Then, replace the *oidc* settings in `.samples.config.json` to point to your new
-    });
-  }
- 
-  login() {
-    this.authClient.token.getWithRedirect({ responseType: 'code' });
-  }
-}
-```
-
-There are a number of different ways to construct the login redirect URL.
-
-1. Build the URL manually
-2. Use an OpenID Connect / OAuth 2.0 middleware library
-3. Use [AuthJS](http://developer.okta.com/code/javascript/okta_auth_sdk)
-
-In this sample, we use AuthJS to create the URL and perform the redirect. An `OktaAuth` object is instantiated with the configuration in `.samples.config.json`. When the `login()` function is called from the view, it calls the [`/authorize`](http://developer.okta.com/docs/api/resources/oauth2.html#authentication-request) endpoint to start the [Authorization Code Flow](https://tools.ietf.org/html/rfc6749#section-1.3.1).
-
-You can read more about the `OktaAuth` configuration options here: [OpenID Connect with Okta AuthJS SDK](http://developer.okta.com/code/javascript/okta_auth_sdk#social-authentication-and-openid-connect).
-
-**Important:** When the authorization code is exchanged for an `access_token` and/or `id_token`, the tokens **must** be [validated](#validation). We'll cover that in a bit.
-
-### Custom Login Form
-To render the [Okta Sign-In Widget](http://developer.okta.com/code/javascript/okta_sign-in_widget), include a container element on the page for the widget to attach to:
-
-```html
-<!-- overview.mustache -->
-<div id="sign-in-container"></div>
-```
-
-Then, initialize the widget with the [OIDC configuration](https://github.com/okta/okta-signin-widget#openid-connect) options:
-``` javascript
-// login-custom.controller.js
-class LoginCustomController {
-  constructor(config) {
-    this.config = config;
-  }
- 
-  $onInit() {
-    const signIn = new SignIn({
-      baseUrl: this.config.oktaUrl,
-      clientId: this.config.clientId,
-      redirectUri: this.config.redirectUri,
-      authParams: {
-        issuer: this.config.issuer,
-        responseType: 'code',
-        scopes: ['openid', 'email', 'profile'],
-      },
-    });
-    signIn.renderEl({ el: '#sign-in-container' }, () => {});
-  }
-}
-```
-To perform the [Authorization Code Flow](https://tools.ietf.org/html/rfc6749#section-1.3.1), we set the `responseType` to `code`. This returns an `access_token` and/or `id_token` through the [`/token`](http://developer.okta.com/docs/api/resources/oauth2.html#token-request) OpenID Connect endpoint.
+### Login Redirect (using [okta oidc-middleware](https://github.com/okta/okta-oidc-js/tree/master/packages/oidc-middleware))
 
-**Note:** Additional configuration for the `SignIn` object is available at [OpenID Connect, OAuth 2.0, and Social Auth with Okta](https://github.com/okta/okta-signin-widget#configuration).
+> Note: The middleware library [okta oidc-middleware](https://github.com/okta/okta-oidc-js/tree/master/packages/oidc-middleware) is used by the Express back-end to handle the [Authorization Code Flow](https://tools.ietf.org/html/rfc6749#section-1.3.1) 
```

Comment:
nit: We link to this lib in the header too. I'd probably remove the link in the header and keep this.

Reply:
done